### PR TITLE
支持按网关拉取请求量数据

### PR DESCRIPTION
**Pull request by alex-smile**
### Description

Fixes # (issue)

### Checklist

- [ ] 填写 PR 描述及相关 issue (write PR description and related issue)
- [ ] 代码风格检查通过 (code style check passed)
- [ ] PR 中包含单元测试 (include unit test)
- [ ] 单元测试通过 (unit test passed)
- [ ] 本地开发联调环境验证通过 (local development environment verification passed)

### CODE REVIEW

File: src/dashboard/apigateway/apigateway/apps/metrics/statistics.py [hunk 1]
```diff
@@ -41,60 +41,54 @@ def __init__(self):
...
-
         # 统计请求数/失败请求数
-        api_request_data = defaultdict(dict)
-        for item in api_request_count["series"]:
+        gateway_name_to_request_data: Dict = defaultdict(dict)
+        for item in gateway_request_count["series"]:
             dimensions = item["dimensions"]
 
-            gateway_name = dimensions["api_name"]
+            gateway_name_ = dimensions["api_name"]
```
Comment (wklken) [inline src/dashboard/apigateway/apigateway/apps/metrics/statistics.py:77]: 换个名字或者`_`加到前面, 放后面很奇怪

File: src/dashboard/apigateway/apigateway/apps/metrics/statistics.py [hunk 2]
```diff
@@ -41,60 +41,54 @@ def __init__(self):
...
-
-            if gateway_name in api_request_data and key in api_request_data[gateway_name]:
-                api_request_data[gateway_name][key]["total_msecs"] = item["datapoints"][0][0]
+                gateway_name_to_request_data[gateway_name_][key]["failed_count"] += count
 
         # 保存数据
         statistics_record = []
-        for gateway_name, gateway_request_data in api_request_data.items():
-            gateway_id = self._get_gateway_id(gateway_name)
+        for gateway_name_, gateway_request_data in gateway_name_to_request_data.items():
```
Comment (wklken) [inline src/dashboard/apigateway/apigateway/apps/metrics/statistics.py:88]: 同上

**Review by wklken — APPROVED** [review]

**Comment by wklken** [comment]
单测挂了

File: src/dashboard/apigateway/apigateway/apps/metrics/statistics.py (hunk 1 above)
Comment (alex-smile) [reply src/dashboard/apigateway/apigateway/apps/metrics/statistics.py]: done

**Review by Han-Ya-Jun — APPROVED** [review]
LGTM

**Pull request merged**